modules/shared_data/src/main/scala/org/elpaca_metagraph/shared_data/app/ApplicationConfigOps.scala

```diff
@@ -1,16 +1,21 @@
 package org.elpaca_metagraph.shared_data.app
 
 import cats.effect.kernel.Sync
+import cats.implicits.catsSyntaxEitherId
```

Comment:
please don't use `cats.implicits`. use `cats.syntax` instead. see cats documentation for details